scripts/bozzcoin.coffee

```diff
@@ -25,9 +28,22 @@ module.exports = (robot) ->
       when "run" then return 50
       when "racket steps" then return 1 / 18
       else return 0
+  
+  # returns true if bozziplier is reset
+  updateBozziplier = (username) ->
+    if username is "rurouni"
+      robot.brain.set("bozzLastExercised", new Date())
+      return true
+    else
+      bozzLastExercised = robot.brain.get("bozzLastExercised")
+      if !bozzLastExercised || new Date().getTime() - new Date(bozzLastExercised).getTime() > bozziplierThreshold * 86400000 # threshold times milliseconds in a day
+        bozziplier = robot.brain.get("bozziplier")
+        newBozziplier = bozziplier * 0.5
```

Comment:
0.5 is really harsh, that decays to 0.007 in 1 week.

Perhaps use 0.9? That's 0.478 after 1 week.